libc/src/time/time_utils.h

```diff
@@ -156,6 +156,13 @@ LIBC_INLINE struct tm *gmtime_internal(const time_t *timer, struct tm *result) {
   return result;
 }
 
+// TODO: localtime is not yet implemented and a temporary solution is to
+//       use gmtime, https://github.com/llvm/llvm-project/issues/107597
+LIBC_INLINE struct tm *localtime(const time_t *timer, struct tm *result) {
+  struct tm *result = {0};
```

Comment:
This one shadows the second input parameter?

Reply:
`localtime` should only have one argument, so I removed `result`.